src/Client/Core/Entities/EntityQuery.cs

```diff
@@ -61,9 +61,18 @@ public record EntityQuery
     public bool IncludeState { get; init; } = true;
 
     /// <summary>
-    /// Gets the size of each page to return.
+    /// Gets a value indicating whether to include metadata about entities that have no user-defined state. Defaults to false.
     /// </summary>
-    public int PageSize { get; init; } = DefaultPageSize;
+    /// <remarks> Stateless entities occur when the storage provider is tracking metadata about an entity for synchronization purposes
+    /// even though the entity does not "logically" exist, in the sense that it has no application-defined state.
+    /// Stateless entities are usually transient. For example, they may be in the process of being created or deleted, or they may have been locked by a critical section.
+    /// </remarks>
+    public bool IncludeStateless { get; init; }
```

Comment:
Not sure that I prefer an enum since the two are logically independent (all four combinations are possible), and one of them is going to be relevant for most queries (IncludeState) while the other is an obscure feature likely to never be touched by most users (IncludeStateless).

Reply:
I have renamed IncludeStateless to IncludeTransient as discussed.